airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/shared/ActivityConfiguration.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling.shared;
+
+import io.airbyte.config.EnvConfigs;
+import io.airbyte.workers.temporal.TemporalUtils;
+import io.temporal.activity.ActivityCancellationType;
+import io.temporal.activity.ActivityOptions;
+import java.time.Duration;
+
+public class ActivityConfiguration {
```

Comment:
can we add a short doc string here too?

Reply:
Done

Reply:
nit: is this better in TemporalUtils? I see some workflow options helper methods there.